tensorflow/contrib/cmake/tf_core_kernels.cmake

```diff
@@ -92,6 +92,8 @@ if(tensorflow_BUILD_CONTRIB_KERNELS)
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/kernels/nccl_manager.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/kernels/nccl_ops.cc"
       "${tensorflow_source_dir}/tensorflow/contrib/nccl/ops/nccl_ops.cc"
+      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/kernels/periodic_resample_op.cc"
+      "${tensorflow_source_dir}/tensorflow/contrib/periodic_resample/ops/array_ops.cc"
```

Comment:
(I just commented about this on the closed PR...) Revert these two lines and the corresponding changes on L166–167. These are basically adding two elements to a list and removing them again :). Since you're building an `AddUserOps` library in `tf_python.cmake`, you don't need any changes in this file.

Reply:
Fixed!